### DISCUSSION TITLE
Suggestion for changing the UI terminology

### DISCUSSION THREAD

**Discussion by DharmaDischarge** [mastodon/mastodon#21969] [General]
I wanted to suggest changing
"Follow" to "Listen to?" and "Following" to "Listening."
and "Unfollow" to "Walk away."
It would make us more aware morally and feel culpable for our engagement.

For example, if someone is spewing hate by us presenting it as a question of "Listen to?" I feel it would at least make people think about their choices on Mastodon and remind us of our moral obligation to each other as human beings.

I also feel it aligns with the etiquette of the app during my short time with it.

And it would be easy to implement from what I have been told.

The only downside I can think of is it would confuse users who are used to the current setup.

But the benefits of distinguishing different values from other social media I think outweigh the growing pains.

**Comment by edoswald** [comment]
Given my interactions with folks on Mastodon, I don't think this is necessary. While I see what you're saying, I think it overcomplicates Mastodon even more. It's already an adjustment for folks not used to how things were before Facebook and Twitter and MySpace. I think using similar language eases the transition into a very different world for many. Maybe down the road if it's necessary, but i do believe we're doing well now.

**Reply by DharmaDischarge** [reply]
I understand and thank you. the only thing I can add is I am afraid it will become more like Twitter with an influx of new users who have been conditioned from a decade of desensitization using social media as an escape from the standards of the real world. We thought it wouldn't bleed into our daily lives. I know it sounds a bit melodramatic of me but I am afraid this could our last chance for a culture change of kindness before we live in a literal dystopia. though I hope I am wrong. 

But, don't get me wrong I love Mastodon I have seven social anxiety and some other diagnosis that make it almost impossible for me to socialize on an in-person level.  It has been life-changing for me. I just know the users of the bird site who are still there are ingrained in that culture which is why they haven't left in spite of the near madness.

though if you all will keep it on the table if things do change for the worst then I am happy.
